.eslintrc.js

```diff
@@ -1,10 +1,27 @@
-/**
```

Comment:
looks like you did this whole PR in one commit, is that right?  If so, try breaking your work up into a commit per idea, such as a function.  It's easier to follow and best practice to my understanding.

Reply:
Yes, I just realized this now. I will do that from now on.